ocs_ci/ocs/uninstall.py

```diff
@@ -212,14 +226,85 @@ def uninstall_ocs():
             .get("storageClassName")
         )
 
+    # step 6: delete storagecluster
+    cleanup_policy = (
+        storage_cluster.get()
+        .get("metadata")
+        .get("annotations")
+        .get("uninstall.ocs.openshift.io/cleanup-policy")
+    )
+
     log.info("Deleting storageCluster object")
     storage_cluster.delete(resource_name=constants.DEFAULT_CLUSTERNAME)
 
+    # step 7+9: check cleanup pods are compleated
+    storage_node_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    if cleanup_policy == "delete":
+        # check clean up pods
+        cleanup_pods = [
+            pod for pod in get_all_pods() if "cluster-cleanup-job" in pod.name
+        ]
+        for pod in cleanup_pods:
+            if pod.get().get("status").get("phase") == "Succeeded":
+                log.info(f"Cleanup pod {pod.name} completed successfully ")
+            else:
+                log.error(f"Cleanup pod {pod.name} did not complete")
+        # confirm var/lib/rook is deleted
+        for node in storage_node_list:
+            try:
+                assert not ocp_obj.exec_oc_debug_cmd(
+                    node=node, cmd_list=["ls -l var/lib/rook"]
+                ), "OCS artificats were not deleted from nodes "
+            except CommandFailed:
```

Comment:
why should we catch and continue if the command failed?

Reply:
forgot to throw an error, added pass just to be able to commit :)